docs/spec/slashing/end_block.md

```diff
@@ -1,21 +1,21 @@
-# End-Block 
+# End-Block
 
 ## Slashing
 
 Tendermint blocks can include
 [Evidence](https://github.com/tendermint/tendermint/blob/develop/docs/spec/blockchain/blockchain.md#evidence), which indicates that a validator
 committed malicious behaviour. The relevant information is forwarded to the
 application as [ABCI
-Evidence](https://github.com/tendermint/abci/blob/develop/types/types.proto#L259), so the validator an be accordingly punished.
+Evidence](https://github.com/tendermint/tendermint/abci/blob/develop/types/types.proto#L259), so the validator an be accordingly punished.
```

Comment:
same deal:

https://github.com/tendermint/tendermint/blob/develop/abci/types/types.proto#L259

is the correct link